blackbox/docs/sql/reference/copy_from.txt

```diff
@@ -41,7 +41,8 @@ For examples see: :ref:`importing_data`.
   When importing data using ``COPY FROM`` Crate does not check if the types
   from the columns and the types from the import file match. It does not cast
   the types to their target but will always import the data as it is
-  in the source file.
+  in the source file. Crate does not also check for null values for columns
```

Comment:
Maybe formulate this in a more generic way that column constraints aren't checked?